allauth/socialaccount/providers/metamask/templates/metamask/auth.html

```diff
@@ -0,0 +1,73 @@
+{% load static %}
+<div id="personalSign">  </div>
+<HR>
+<div id="personalSignResult">  </div>
+<HR>
+<script type="text/javascript">
+if (typeof jQuery == 'undefined') {
+  document.write('<script type="text/javascript" src="https://code.jquery.com/jquery-3.6.0.min.js"\n' +
+    '  integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4="\n' +
+    '  crossorigin="anonymous"><\/script>');
+  }
+</script>
+<script src="https://cdn.jsdelivr.net/npm/web3@latest/dist/web3.min.js"></script>
+<script type="text/javascript">
```

Comment:
It would be best to move this JS out of the .html -- you can have a look at `fbconnect.html` to see how it is done there.

Reply:
Are you talking about the CDN include? fbconnect.html does something similar except with a locally hosted file - I didn't want to tie myself to that because web3 is evolving a lot right now. I actually looked at fbconnect.html when building this because I wanted to see how other providers handled javascript includes.

Reply:
No, I was referring to the additional JS inlined in this file... FB has it in `allauth/socialaccount/providers/facebook/static/facebook/js/fbconnect.js`

Reply:
Fixed this as well

Reply:
Latest PR to my branch fixes web3 dependency